packages/create-waku/src/index.ts

```diff
@@ -202,15 +203,33 @@ async function init() {
     await installTemplate(root, packageName, templateRoot, templateName);
   }
 
-  // TODO automatically installing dependencies
   // 1. check packageManager
   // 2. and then install dependencies
-
-  console.log(`\nDone. Now run:\n`);
-  console.log(`${bold(green(`cd ${targetDir}`))}`);
-  console.log(`${bold(green(commands.install))}`);
-  console.log(`${bold(green(commands.dev))}`);
   console.log();
+  console.log(`Installing dependencies by running ${commands.install}...`);
+
+  process.chdir(targetDir);
```

Comment:
This changes the current working directory and won't go back.

Should we use `cwd` option in `spawn`?

Reply:
> This changes the current working directory and won't go back.

Because of this, what do you concern with?

Reply:
a) I don't like this side-effecty behavior
b) The message at L228 is wrong.

Reply:
1ad5c5d

I got your thought! check this commit :)